h/views/api/helpers/media_types.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from h.views.api import API_VERSIONS
+
+
+def media_type_for_version(version, subtype="json"):
+    """
+    Return the media type corresponding to a particular version string
+
+    :arg version:  The major API version, e.g. ``"v1"``
+    :type version: str
+    :arg subtype:  The "subtype" of the content type desired; defaults
+                   to ``"json"``
+    :type subtype: str or None
+    :rtype: str
+    """
+    return "application/vnd.hypothesis.{}+{}".format(version, subtype)
+
+
+def valid_media_types(versions=None):
```

Comment:
Taking a `versions` arg here is pretttttty much entirely for dependency injection for tests.